larsim/Simulation/simulationservices.fcl

```diff
@@ -42,7 +43,17 @@ standard_largeantparameters:
  ModBoxA: 0.930
  ModBoxB: 0.212
 
- # The following parameters specify details of wireplanes or similar
+ #* Recombination factor coefficients for LArQL can be found in https://drive.google.com/file/d/1zG5lLqMxACcvuJqQXhLnQ8Cnmz2YEZPU/view?usp=sharing
```

Comment:
are the coefficients searchable elsewhere?  This comment relies on the googledrive link remaining valid...